test/features/user_records_an_application_test.rb

```diff
@@ -42,18 +46,20 @@ def test_viewing_the_details_of_an_application
   end
 
   def test_editing_the_details_of_an_application
-    app = Application.create!(:company => "Basecamp",
+    user = create(:person)
+    app = user.applications.create!(:company => "Basecamp",
                               :url => "http://basecamp.com",
                               :location => "Chicago, IL",
-                              :status => "open")
+                              :status => "applied")
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.